### PR TITLE
correct CS

### PR DESCRIPTION
**Pull request by sstok**
|Q            |A  |
|---          |---|
|Fixed Tickets|   |
|License      |MIT|

### CODE REVIEW

**Comment by sstok** [comment]
(PR squashed)

**Comment by sstok** [comment]
(PR squashed)

**Comment by sstok** [comment]
Strange why is StyleCI failing after I squashed the PR :|

File: README.md [hunk 1]
```diff
@@ -5,7 +5,7 @@
 [![Scrutinizer Quality Score](https://scrutinizer-ci.com/g/gushphp/gush/badges/quality-score.png?s=127d28d94969ef366d3bc78808cc89b8eeba51e2)](https://scrutinizer-ci.com/g/gushphp/gush/)
 [![Latest Stable Version](https://poser.pugx.org/gushphp/gush/v/stable.png)](https://packagist.org/packages/gushphp/gush) [![Total Downloads](https://poser.pugx.org/gushphp/gush/downloads.png)](https://packagist.org/packages/gushphp/gush) [![Latest Unstable Version](https://poser.pugx.org/gushphp/gush/v/unstable.png)](https://packagist.org/packages/gushphp/gush)
 [![Dependency Status](https://www.versioneye.com/php/gushphp:gush/badge.svg)](https://www.versioneye.com/php/gushphp:gush/)
-[![StyleCI](https://styleci.io/repos/16789051/shield?style=flat)](https://styleci.io/repos/16789051)
+[![StyleCI](https://styleci.io/repos/16789051/shield)](https://styleci.io/repos/16789051)
```
Comment (GrahamCampbell) [inline README.md]: this conflict resolution looks wrong

Comment (sstok) [inline README.md]: Yea :) I forgot to rebase...

**Comment by sstok** [comment]
(PR squashed)

**Comment by GrahamCampbell** [comment]
All failures are header comment related now.

**Comment by sstok** [comment]
Hmm, it seems the pull-request:squash command is a bit broken :|

**Comment by sstok** [comment]
Ah the `pull-request:squash` command uses the local branch! Thats something I need to fix.

**Comment by cordoval** [comment]
did you fix the squash command?

File: src/Subscriber/GitRepoSubscriber.php [hunk 2]
```diff
@@ -291,7 +291,7 @@ private function initializeIssueTracker(ConsoleCommandEvent $event, $org, $repo,
 
         $issueTracker->authenticate();
 
-        /** @var \Gush\Adapter\BaseIssueTracker $issueTracker */
+        /* @var \Gush\Adapter\BaseIssueTracker $issueTracker */
```
Comment (GrahamCampbell) [inline src/Subscriber/GitRepoSubscriber.php]: this change looks wrong

**Comment by sstok** [comment]
> did you fix the squash command?

It's a bit more complex to fix, I will make a new issue with some notes on how to fix this.

File: src/Subscriber/GitRepoSubscriber.php (hunk 2 above)
Comment (GrahamCampbell) [inline src/Subscriber/GitRepoSubscriber.php]: This phpdoc should be either deleted, or moved to the  actual variable assignment.

**Comment by cordoval** [comment]
woah! i merged thinking this was over

File: src/Subscriber/GitRepoSubscriber.php (hunk 2 above)
Comment (GrahamCampbell) [inline src/Subscriber/GitRepoSubscriber.php]: See https://github.com/sstok/gush/pull/6.

Comment (cordoval) [inline src/Subscriber/GitRepoSubscriber.php]: yeah sorry about the early merge, ok let's fix this then

**Comment by GrahamCampbell** [comment]
See https://github.com/gushphp/gush/pull/482.

**Pull request merged**